internal/installutil/k8s/k8s.go

```diff
@@ -0,0 +1,82 @@
+package k8s
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/waypoint/internal/clierrors"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type K8sConfig struct {
+	serverImage        string            `hcl:"server_image,optional"`
+	namespace          string            `hcl:"namespace,optional"`
+	serviceAnnotations map[string]string `hcl:"service_annotations,optional"`
+
+	odrImage              string `hcl:"odr_image,optional"`
+	odrServiceAccount     string `hcl:"odr_service_account,optional"`
+	odrServiceAccountInit bool   `hcl:"odr_service_account_init,optional"`
+
+	advertiseInternal bool   `hcl:"advertise_internal,optional"`
+	imagePullPolicy   string `hcl:"image_pull_policy,optional"`
+	k8sContext        string `hcl:"k8s_context,optional"`
+	openshift         bool   `hcl:"openshft,optional"`
```

Comment:
Unless this needs to be used, can we remove it?

Reply:
Oh good catch, I think I just missed this in my rebase!

Reply:
Resolved in https://github.com/hashicorp/waypoint/commit/36e943e2457c4eac4a0cedcc2e2c79f82a5d0eb1